packages/lib/src/components/internal/SecuredFields/SecuredFieldsProviderHandlers.ts

```diff
@@ -119,9 +119,15 @@ function handleOnBrand(cardInfo: CbObjOnBrand): void {
  * Handles validation errors
  */
 function handleOnError(cbObj: CbObjOnError, hasUnsupportedCard: boolean = null): boolean {
+    console.log('\n### SecuredFieldsProviderHandlers::handleOnError:: this.state.hasUnsupportedCard', this.state.hasUnsupportedCard);
```

Comment:
Is this intentional? Shall we remove them or comment them with `process.env` conitional?

Reply:
No, it's not. I thought I'd removed all logs

Reply:
fixed